addons/core/functions/fnc_initKeybinds.sqf

```diff
@@ -114,56 +114,63 @@ TF_speak_volume_modifier_yelling_modifiers = [false, false, false];
+["TFAR","LRChannel8",8 call _fnc_localizeLRChannel,{[7] call TFAR_fnc_processLRChannelKeys},{true},[TF_lr_channel_8_scancode, TF_lr_channel_8_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","LRChannel9",9 call _fnc_localizeLRChannel,{[8] call TFAR_fnc_processLRChannelKeys},{true},[TF_lr_channel_9_scancode, TF_lr_channel_9_modifiers],false] call cba_fnc_addKeybind;
+
+
+["TFAR","ChangeSpeakingVolume",["STR_TFAR_key_ChangeSpeechVolume","STR_TFAR_key_ChangeSpeechVolume"],{call TFAR_fnc_onSpeakVolumeChangePressed},{true},[TF_speak_volume_scancode, TF_speak_volume_modifiers],false] call cba_fnc_addKeybind;
+
+["TFAR", "YellingModifier", ["STR_TFAR_key_YellingModifier", "STR_TFAR_key_YellingModifier"], {["yelling"] call TFAR_fnc_onSpeakVolumeModifierPressed}, {call TFAR_fnc_onSpeakVolumeModifierReleased}, [TF_speak_volume_modifier_yelling_scancode, TF_speak_volume_modifier_yelling_modifiers], false] call cba_fnc_addKeybind;
+["TFAR", "WhisperingModifier", ["STR_TFAR_key_WhisperingModifier", "STR_TFAR_key_WhisperingModifier"], {["whispering"] call TFAR_fnc_onSpeakVolumeModifierPressed}, {call TFAR_fnc_onSpeakVolumeModifierReleased}, [TF_speak_volume_modifier_whispering_scancode, TF_speak_volume_modifier_whispering_modifiers], false] call cba_fnc_addKeybind;
+
+["TFAR","CycleSWRadios",["STR_TFAR_key_CycleRightSWRadios","STR_TFAR_key_CycleRightSWRadios"],{true},{["next"] call TFAR_fnc_processSWCycleKeys},[TF_sw_cycle_next_scancode, TF_sw_cycle_next_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","CycleSWRadios",["STR_TFAR_key_CycleLeftSWRadios","STR_TFAR_key_CycleLeftSWRadios"],{true},{["prev"] call TFAR_fnc_processSWCycleKeys},[TF_sw_cycle_prev_scancode, TF_sw_cycle_prev_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","CycleLRRadios",["STR_TFAR_key_CycleRightLRRadios","STR_TFAR_key_CycleRightLRRadios"],{true},{["next"] call TFAR_fnc_processLRCycleKeys},[TF_lr_cycle_next_scancode, TF_lr_cycle_next_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","CycleLRRadios",["STR_TFAR_key_CycleLeftLRRadios","STR_TFAR_key_CycleLeftLRRadios"],{true},{["prev"] call TFAR_fnc_processLRCycleKeys},[TF_lr_cycle_prev_scancode, TF_lr_cycle_prev_modifiers],false] call cba_fnc_addKeybind;
+
+
+["TFAR","SWStereoBoth",    ["STR_TFAR_key_SWStereo_Both","STR_TFAR_key_SWStereo_Both"],{[0] call TFAR_fnc_processSWStereoKeys},    {true},[TF_sw_stereo_both_scancode, TF_sw_stereo_both_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","SWStereoLeft",    ["STR_TFAR_key_SWStereo_Left","STR_TFAR_key_SWStereo_Left"],{[1] call TFAR_fnc_processSWStereoKeys},    {true},[TF_sw_stereo_left_scancode, TF_sw_stereo_left_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","SWStereoRight",["STR_TFAR_key_SWStereo_Right","STR_TFAR_key_SWStereo_Right"],{[2] call TFAR_fnc_processSWStereoKeys},{true},[TF_sw_stereo_right_scancode,TF_sw_stereo_right_modifiers],false] call cba_fnc_addKeybind;
```

Comment:
What happened to the indentation here? :D

Reply:
Ha, don't know XD 
I fixed this.